recipes/dnmtools/meta.yaml

```diff
@@ -5,7 +5,9 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 1
+  run_exports:
+    - {{ pin_subpackage('dnmtools', max_pin="x.x") }}
```

Comment:
Is a CLI, ABI or API breakage expected in new minor versions? Please indicate this with a short comment here. Otherwise, please pin to the major version (`max_pin="x"`).

Reply:
@johanneskoester Thanks for the tip. I will pin the major version.